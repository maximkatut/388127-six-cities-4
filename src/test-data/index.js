export const offers = [
  {
    appliences: [`Wi-Fi`, `Microwave`],
    bedrooms: 3,
    coordinates: [52.3809553943508, 4.939309666406198],
    description: [`A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.`, `An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`],
    guests: 2,
    host: {
      avatar: `img/avatar-angelina.jpg`,
      name: `Max`,
      super: true
    },
    id: `1`,
    imagesGallery: [
      `img/apartment-01.jpg`,
      `img/apartment-02.jpg`,
      `img/apartment-03.jpg`,
      `img/room.jpg`
    ],
    mainImage: `img/apartment-01.jpg`,
    offerType: `House`,
    premium: true,
    price: 200,
    rate: 5,
    title: `Nice, cozy, warm big bed house`
  },
  {
    appliences: [`Wi-Fi`, `Microwave`],
    bedrooms: 4,
    coordinates: [52.3809553943508, 4.939309666406198],
    description: [`An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`],
    guests: 3,
    host: {
      avatar: `img/avatar-max.jpg`,
      name: `Anton`,
      super: false
    },
    id: `2`,
    imagesGallery: [
      `img/apartment-01.jpg`,
      `img/apartment-03.jpg`,
      `img/room.jpg`
    ],
    mainImage: `img/apartment-02.jpg`,
    offerType: `Apartment`,
    premium: true,
    price: 300,
    rate: 4,
    title: `Nice, cozy, warm big bed apartment`
  }
];

