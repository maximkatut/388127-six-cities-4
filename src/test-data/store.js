import {cities, offers} from '.';
import configureStore from 'redux-mock-store';

const mockStore = configureStore([]);

export const store = mockStore({
  activeCityName: cities[0].name,
  offers,
  cities
});
