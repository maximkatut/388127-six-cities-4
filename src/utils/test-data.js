export const offerCards = [
  {
    id: `1`,
    imageSrc: `img/apartment-02.jpg`,
    offerType: `Room`,
    premium: false,
    price: 200,
    rate: 5,
    title: `Nice, cozy, warm big bed apartment`
  },
  {
    id: `2`,
    imageSrc: `img/apartment-03.jpg`,
    offerType: `House`,
    premium: true,
    price: 300,
    rate: 4,
    title: `Nice, cozy, warm big bed apartment`
  }
];
