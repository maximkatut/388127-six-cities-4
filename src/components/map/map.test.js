import Map from './map.jsx';
import React from 'react';
import {offers, cities} from '../../test-data';
import renderer from 'react-test-renderer';

describe(`Map`, () => {
  it(`Map component should render correctly`, () => {
    const tree = renderer.create(
        <Map
          offers={offers}
          cities={cities}
        />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

