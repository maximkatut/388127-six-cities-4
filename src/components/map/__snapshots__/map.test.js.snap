// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Map component should render correctly 1`] = `
<section
  className="cities__map map"
/>
`;
